aten/src/ATen/native/cuda/Reduce.cuh

```diff
@@ -336,7 +333,7 @@ struct ReduceJitOp {
   }
 };
 
-template <typename scalar_t, typename ops_t, typename index_t, typename out_scalar_t=scalar_t, int vt0=4>
+  template <typename scalar_t, typename ops_t, typename index_t, typename out_scalar_t=scalar_t, int vt0=4, int input_vec_size=vt0>
```

Comment:
nit: why the whitespace change?

```suggestion
template <typename scalar_t, typename ops_t, typename index_t, typename out_scalar_t=scalar_t, int vt0=4, int input_vec_size=vt0>
```

Reply:
Fixed and merged against latest pytorch/main.